client/keystore/src/local.rs

```diff
@@ -68,6 +68,9 @@ impl LocalKeystore {
 	}
 }
 
+/// Implementation details:
+/// - Key generation functions (e.g. `ed25519_generate_new`): if the `seed` parameter is `Some(_)`
+///   then the key will be ephemeral and stored in memory.
```

Comment:
nit: I guess this would be better as doc comments for the actual `*_generate_new()` functions